evolvedbinary/jdita
# Fix `eslint` configuration.

### Purpose of this PR

This PR is addressing linting errors caused by TS type warnings and has been created as a follow-up task from PR https://github.com/evolvedbinary/jdita/pull/136 in comment: https://github.com/evolvedbinary/jdita/pull/136/files/fdae45a2b3d45a68cd7f9b4f39d3cec7dcb34a57#r1551359116

### How to test this PR

Run `yarn lint` and check the result. 

It should result in 3 warnings for unused-vars in jdita/packages/lwdita-xdita/visitor.ts, which I have left untouched on purpose, because @marmoure will address this in a follow-up fix and the warning will become obsolete.
